src/ai/h2o/ci/BuildInfo.groovy

```diff
@@ -18,19 +18,27 @@ class BuildInfo implements Serializable {
 
     private String version
 
+    final private boolean prJob
+
+    final private boolean triggeredByTimer
```

Comment:
This is good idea, but it would be better to have triggerReason as enum (e.g., {manual, timer, external})